<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Generate_Dechirp">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 127);</string>
       </property>
       <property name="text">
        <string> Dechirp Generation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>320</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadInterference">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>Load Interference Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MovingArmData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>Load Moving Arm Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ReferenceArm">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>Load Reference Arm Data</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OCT_Parameters">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 127);</string>
       </property>
       <property name="text">
        <string>OCT Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
